utils/meta-intel-sdo/recipes-connectivity/ccodesdk/ccodesdk.bb

```diff
@@ -0,0 +1,61 @@
+#sdo yocto project
+
+DESCRIPTION = "Recipe for sdo (c-code-sdk) on linux"
```

Comment:
need to rename the files are clientsdk.bb and need to remove the ref to c-code-sdk.

Reply:
done